docs/csharp/language-reference/keywords/formatting-numeric-results-table.md

```diff
@@ -1,32 +1,54 @@
-  
- For more information about standard and custom formatting strings, see [Formatting Types](../../../standard/base-types/formatting-types.md).
-  
-|Format Specifier|Description|Examples|Output|  
+# Formatting numeric results table (C# Reference)
+
+The following table shows supported format specifiers for formatting numeric results. The formatted result corresponds to the "en-US" <xref:System.Globalization.CultureInfo>.
+
+|Format specifier|Description|Examples|Result|  
 |----------------------|-----------------|--------------|------------|  
-|C or c|Currency|Console.Write("{0:C}", 2.5);<br /><br /> Console.Write("{0:C}", -2.5);|$2.50<br /><br /> ($2.50)|  
-|D or d|Decimal|Console.Write("{0:D5}", 25);|00025|  
-|E or e|Scientific|Console.Write("{0:E}", 250000);|2.500000E+005|  
-|F or f|Fixed-point|Console.Write("{0:F2}", 25);<br /><br /> Console.Write("{0:F0}", 25);|25.00<br /><br /> 25|  
-|G or g|General|Console.Write("{0:G}", 2.5);|2.5|  
-|N or n|Number|Console.Write("{0:N}", 2500000);|2,500,000.00|  
-|X or x|Hexadecimal|Console.Write("{0:X}", 250);<br /><br /> Console.Write("{0:X}", 0xffff);|FA<br /><br /> FFFF|  
-  
-## See Also
-
-- [C# Reference](../../../csharp/language-reference/index.md)  
-- [C# Programming Guide](../../../csharp/programming-guide/index.md)  
-- [Standard Numeric Format Strings](../../../standard/base-types/standard-numeric-format-strings.md)  
-- [Reference Tables for Types](../../../csharp/language-reference/keywords/reference-tables-for-types.md)  
-- [string](../../../csharp/language-reference/keywords/string.md)
+|C or c|Currency|`string s = $"{2.5:C}";`<br /><br /> `string s = $"{-2.5:C}";`|$2.50<br /><br /> ($2.50)|  
+|D or d|Decimal|`string s = $"{25:D5}";`|00025|  
+|E or e|Exponential|`string s = $"{250000:E2}";`|2.50E+005|  
+|F or f|Fixed-point|`string s = $"{2.5:F2}";`<br /><br /> `string s = $"{2.5:F0}";`|2.50<br /><br /> 3|  
+|G or g|General|`string s = $"{2.5:G}";`|2.5|  
+|N or n|Numeric|`string s = $"{2500000:N}";`|2,500,000.00|  
+|P or p|Percent|`string s = $"{0.25:P}";`|25.00%|  
+|R or r|Round-trip|`string s = $"{2.5:R}";`|2.5|  
+|X or x|Hexadecimal|`string s = $"{250:X}";`<br /><br /> `string s = $"{0xffff:X}";`|FA<br /><br /> FFFF|  
+
+## Remarks
+
+You use a format specifier to create a format string. The format string is of the following form: `Axx`, where
+
+- `A` is the format specifier, which controls the type of formatting applied to the numeric value.
+- `xx` is the precision specifier, which affects the number of digits in the formatted output. The value of the precision specifier ranges from 0 to 99.
+
+Decimal and hexadecimal format specifiers are supported only for integral types. The round-trip format specifier is supported only for <xref:System.Single>, <xref:System.Double>, and <xref:System.Numerics.BigInteger> types.
```

Comment:
Sorry, @pkulikov. One thing I forgot to mention is that beginning the first sentence with "The" makes it clear that it refers to the specific decimal and hexadecimal specifiers. To even further make that clearer, it's helpful to include the specifiers themselves (e.g., "Decimal ("D" or "d")"). Otherwise, it could be read as a general statement (decimal formatting works only with integer values).